multiverse-source/src/point.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::Result;
+use dcspark_blockchain_source::{
+    cardano::{BlockEvent, CardanoNetworkEvent, Tip},
+    PullFrom,
+};
+
+pub trait Point: PullFrom + Sized + Clone + Send + Sync {
```

Comment:
yeah, this is what I was talking about in https://github.com/dcSpark/carp/pull/116#discussion_r1004481398

if you want to I can move that in this PR and replace it directly. Otherwise I'll do it after this one.

Reply:
#8 made an issue for that, we can do this in next PR